src/lib/include/x86_vector_operators.h

```diff
@@ -0,0 +1,37 @@
+#ifndef DISTOPIA_X86_VECTOR_OPERATORS_H
+#define DISTOPIA_X86_VECTOR_OPERATORS_H
+
+#include "arch_config.h"
+
+#ifdef DISTOPIA_X86_SSE4_1
+
+#include <immintrin.h>
+
+namespace {
+
+#ifdef DISTOPIA_MSVC
```

Comment:
Are these ops MSVC only? I thought most of these are supported by SSE4.1?

Reply:
Other compilers define these operators by default. It’s only MSVC that needs extra help 😊

Reply:
👍